nixpkgs-src.nix

```diff
@@ -0,0 +1,5 @@
+builtins.fetchTarball {
+  # git ls-remote https://github.com/nixos/nixpkgs-channels nixpkgs-unstable
+  url = "https://github.com/nixos/nixpkgs-channels/tarball/f9567594d5af2926a9d5b96ae3bada707280bec6";
+  sha256 = "0vr2di6z31c5ng73f0cxj7rj9vqvlvx3wpqdmzl0bx3yl3wr39y6";
```

Comment:
If a new version of the packages is required, it's as simple as bumping these two lines. `nixpkgs` is the most up to date package manager according to repology &mdash; only second to binary distributions.